<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGoodsToCustomCategoryWidget</class>
 <widget class="QWidget" name="AddGoodsToCustomCategoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{
background-color: rgb(81, 80, 80);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <property name="leftMargin">
       <number>26</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>26</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="CategoryChooseWidget" name="categoryWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>320</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>320</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="GoodsChooseWidget" name="goodsWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>462</width>
          <height>320</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>462</width>
          <height>320</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GoodsChooseWidget</class>
   <extends>QWidget</extends>
   <header>GoodsChooseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CategoryChooseWidget</class>
   <extends>QWidget</extends>
   <header>CategoryChooseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
